packages/ubus-lime-location/files/etc/uci-defaults/93-migrate_libremap_cfg

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/lua
+
```

Comment:
Should we put some kind of time to live on this file, and mark it for removal for some 2022 release?

Reply:
Like the concept. We should open a permanent issue where we collect obsolete code which should be removed with LibreMesh 21.xx release.

Reply:
@G10h4ck please consider listing this file in #809.